src/repositories/account-repository/onbloc-account-repository.ts

```diff
@@ -80,14 +80,31 @@ export class OnblocAccountRepository implements IAccountRepository {
 
     try {
       const response = await this.indexerClient?.pageQuery(
-        makeAccountTransactionsQuery(address, cursor),
+        makeAccountTransactionsQuery(address, cursor, 10_000),
```

Comment:
recommend to use constant variable

Reply:
Added a max query size variable.
https://github.com/onbloc/gnoscan/pull/122/commits/3d1156a1b662f42f660e7755f201184e8ff01b6e